app/assets/stylesheets/application.scss

```diff
@@ -11,8 +11,9 @@
  * It is generally better to create a new file per style scope.
```

Comment:
There are three different `application.*` files for stylesheets. Only `application.scss`should be needed.